chapters/annotations.tex

```diff
@@ -1917,9 +1917,9 @@ \subsection{Version Date and Build Information}\label{version-date-and-build-inf
 \annotationindex{versionDate}\annotationindex{versionBuild}\annotationindex{dateModified}\annotationindex{revisionId}
 
 \begin{example}
-\begin{lstlisting}[language=modelica,mathescape=false]
+\begin{lstlisting}[language=modelica,mathescape=false,escapechar=!]
 package Modelica
-  $\ldots$
+  !$\ldots$!
```

Comment:
Just a side note, the mathmode is not really necessary for `\ldots`

Reply:
Right, why not?  I made the suggested change.  Resolving.